### PR TITLE
feat: Read file details from `Passport()` class

### PR DESCRIPTION
**Pull request by DafyddLlyr**
## What does this PR do?
- Uses a single method to read files from the passport
- Quick refactor before working on the wider file-type changes to add requested files to the ODP schema payload

### CODE REVIEW

File: src/models/passport/passport.ts [hunk 1]
```diff
@@ -1,32 +1,100 @@
 import { get as getByKeyPath, has } from "lodash";
 
+// import { ServiceFiles } from "../../export/digitalPlanning/schema/types";
```
Comment (DafyddLlyr) [inline src/models/passport/passport.ts:3]: These are intentionally commented out to allow this PR to proceed without a schema update - they'll be commented back in shortly.

File: src/export/digitalPlanning/model.ts [hunk 2]
```diff
@@ -716,42 +716,38 @@ export class DigitalPlanning {
...
-                  key,
-                ),
-              });
-            }
-          } catch (err) {
-            throw new Error(`Error formatting files: ${err}`);
-          }
-        });
-      });
+    this.passport.files.forEach(({ url, key }) => {
```
Comment (jessicamcinchak) [inline src/export/digitalPlanning/model.ts:719]: one small question: can we confirm that we'll still pick up location plan files if we read from `files` rather than `data`? I don't see a test case covering that? It's a bit of an edge case as it's the only way to upload a file outside of an upload component.

**Review by jessicamcinchak — APPROVED** [review]
Looks good, one small non-blocking question. 

Thanks for tidy up!

File: src/export/digitalPlanning/model.ts (hunk 2 above)
Comment (DafyddLlyr) [reply src/export/digitalPlanning/model.ts:719]: Very good shout - will add a test case 👍
